os/compression/compress_read.c

```diff
@@ -56,7 +58,7 @@ static struct s_buffer buffers;
  *   Non-negative value on Success.
  *   Negative value on Failure.
  ****************************************************************************/
-static int compress_decompress_block(unsigned char *out_buffer, unsigned int *writesize, unsigned char *read_buffer, unsigned int *size, int index)
+int compress_decompress_block(unsigned char *out_buffer, unsigned int *writesize, unsigned char *read_buffer, unsigned int *size, int index)
```

Comment:
Is this for calling this function from testcase...?
If yes, how about using testcase_drv for testing these functions?

Reply:
Currently the testcase only handles the flat build case. Also I will look into how to use testcase_drv for this

Reply:
I agree with jeongchan, using testcase_drv is a better approach to test this API.

Reply:
Made the changes